<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Encrypter/Decrypter</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#widget{
background-color:qlineargradient(spread:pad, x1:0.051, y1:0.0568182, x2:0.926136, y2:0.938, stop:0 rgba(100, 111, 162, 255), stop:1 rgba(218, 218, 218, 255))}</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>0</y>
      <width>551</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 46pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>File encryption</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Shell Dlg 2&quot;;
color:rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Select file</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>291</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(0, 0, 0);
font: 22pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Select public key</string>
    </property>
   </widget>
   <widget class="QPushButton" name="file_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(219, 212, 255);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_line">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>200</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="submit_button">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>350</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(219, 212, 255);</string>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="arrow_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>75</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 48pt &quot;MS Shell Dlg 2&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>‚Üê</string>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(165, 0, 0);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>290</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
